frontend/packages/kubevirt-plugin/integration-tests/tests/vm.tab.console.rdp.scenario.ts

```diff
@@ -42,7 +42,7 @@ describe('KubeVirt VM console - RDP', () => {
     const vmName = 'windows-rdp';
 
     // for cmd-line scripts only
-    execSync(`kubectl config set-context --current --namespace=${testName}`, { stdio: 'inherit' });
```

Comment:
why was this here?

Reply:
I don't know why it output to stdout, just want to keep the tests output clean if it's not necessay?

Reply:
maybe it was to make clear that we are changing namespace to the active user?

Reply:
I would like to make it quite because a single line output does not help.

Reply:
it could help somebody, but ok, I will leave it to your judgment

Reply:
thanks.